prow/cmd/phaino/local.go

```diff
@@ -278,31 +365,35 @@ func convertToLocal(ctx context.Context, log *logrus.Entry, pj prowapi.ProwJob,
 		return scanln(ctx)
 	}
 
+	// Mount repositories.
 	if decoration != nil {
 		var refs []prowapi.Refs
 		if pj.Spec.Refs != nil {
 			refs = append(refs, *pj.Spec.Refs)
 		}
 		refs = append(refs, pj.Spec.ExtraRefs...)
 		for _, ref := range refs {
-			path := pathAlias(ref)
-			repo, err := readRepo(ctx, path, repoPathMap, readUserInput)
-			if err != nil {
-				return nil, fmt.Errorf("bad repo(%s): %v", path, err)
+			repoPath := pathAlias(ref)
+			dest := filepath.Join(GOSRCPATH, repoPath)
+			// The repo hasn't been mounted.
+			if _, ok := opt.extraVolumesMounts[filepath.Join(GOSRCPATH, repoPath)]; !ok {
```

Comment:
nit: do we also want to support full path like `/go/src/k8s/test-infra` aside from `k8s/test-infra`?

Reply:
Technically user should expect to pass in `/go/src/k8s/test-infra=my/local/clone/path` tbh since this is a volume mount

Reply:
Hmm it was already supported in this way - users are always expected to provide full path via the `--extra-volume-mounts` flag.